goalie/mesh_seq.py

```diff
@@ -16,6 +16,8 @@
 from collections.abc import Iterable
 import numpy as np
 
+comm = firedrake.COMM_WORLD
```

Comment:
Please don't introduce global variables unless absolutely necessary.

Reply:
Fixed in 03af7e015d4a7e59220acc6b6702837c92f0f821.